packages/victory-chart/src/helper-methods.js

```diff
@@ -224,73 +200,18 @@ const getDomain = (props, axis, childComponents) => {
   return invertDomain ? domain.concat().reverse() : domain;
 };
 
-const getAxisOffset = (props, calculatedProps, orientation) => {
-  const { scale, origin, domain, padding } = calculatedProps;
-  const { top, bottom, left, right } = padding;
-  const orientations = {
-    x:
-      orientation === "bottom" || orientation === "top"
-        ? orientation
-        : calculatedProps.orientations.x,
-    y:
-      orientation === "left" || orientation === "right"
-        ? orientation
-        : calculatedProps.orientations.y
-  };
-  // make the axes line up, and cross when appropriate
-  const orientationOffset = {
-    y: orientations.x === "bottom" ? bottom : top,
-    x: orientations.y === "left" ? left : right
-  };
-  const originOffset = {
-    x: orientations.y === "left" ? 0 : props.width,
-    y: orientations.x === "bottom" ? props.height : 0
-  };
-
-  const originPosition = {
-    x: origin.x === domain.x[0] || origin.x === domain.x[1] ? 0 : scale.x(origin.x),
-    y: origin.y === domain.y[0] || origin.y === domain.y[1] ? 0 : scale.y(origin.y)
-  };
-
-  return {
-    x: originPosition.x ? Math.abs(originOffset.x - originPosition.x) : orientationOffset.x,
-    y: originPosition.y ? Math.abs(originOffset.y - originPosition.y) : orientationOffset.y
-  };
-};
-
-const getHorizontalAxisOffset = (props, calculatedProps, orientation) => {
-  const { scale, origin, domain, padding } = calculatedProps;
-  const { top, bottom, left, right } = padding;
+function getOrientation(axis, originSign, horizontal) {
```

Comment:
Are orientations still being used for anything in VictoryChart?

Reply:
I wasn't sure if another type of graph that VictoryChart might wrap used orientation so I left it in.  I searched globally in the project and saw some things that might, but I'd have to really trace through to be sure. I'll look again.

Reply:
Looks like nothing does, I removed it.